assets/js/app/listing/Components/Table/Row/index.vue

```diff
@@ -82,7 +82,17 @@ export default {
     labels: Object,
   },
   computed: {
+    slug() {
+      if (typeof this.record.fieldValues.slug === 'string') {
+        return this.record.fieldValues.slug;
+      }
+      // if slug has different locales, return the 0st one
+      return this.record.fieldValues.slug[
+        Object.keys(this.record.fieldValues.slug)[0]
+      ];
+    },
     size() {
+      console.log(this.record);
```

Comment:
ಠ_ಠ 

Doesn't ESlint trip on that?

Reply:
Oops! Seems like it doesn't... I'll look up if there's an option for it though!